### DISCUSSION TITLE
How do I create an apllication when there is no 'create'-button?

### DISCUSSION THREAD

**Discussion by Kouwe-sudo** [exadel-inc/CompreFace#906] [Q&A]
I installed CompreFace and everything is up and running but I can't use it because I can't create an application. The 'Create'-button is not there. Thus I can't get an API-key and I can't use CompreFace.  
What I do ended up with are two users with the same credentials. One as 'user' and one as 'owner'. I can delete the 'user' but then I'm logged out and when I log back in I find the same two users. 

![CompreFace](https://user-images.githubusercontent.com/59831324/192112211-0ade4829-7ca1-4bf3-af04-d08c38d79617.png)

I think something went wrong but reinstalling CompreFace didn't help (deleting persistent data as well). Is there a way to repair this or a workaround?

**Comment by pospielov** [comment]
https://github.com/exadel-inc/CompreFace/issues/459
Please look at this issue. I described how to clean the database.

I am still wondering how you managed to end up with two users with the same credentials. Could you describe the process so we can reproduce it and fix it?
